.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.rs]
+indent_style = space
+indent_size = 4
+
+[*.toml]
+indent_style = space
+indent_size = 4
```

Comment:
Why do we need to have `[*.rs]` and `[*.toml]` when `[*]` have already provided `indent_style = space` and `indent_size = 4`?

Reply:
Good questons. The answer is most likely cargo cult. I copy this file between repositories, and in some the defaults are 2 spaces, while I still want Rust files to be 4 spaces.

Reply:
Unrelated: Is there a `cargo cult` command?